getting-started/.s2i/environment

```diff
@@ -0,0 +1,6 @@
+JAVA_APP_JAR=*-runner.jar
```

Comment:
NOK, this doesn't actually work (with `registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift`)

Reply:
Very interesting to know, :+1:

Reply:
:-(

Reply:
in my builder I used something like this to compute the actual  build artifact name

```
ARTIFACT_NAME=$(mvn org.apache.maven.plugins:maven-help-plugin:3.1.1:evaluate -Dexpression=project.build.finalName -q -DforceStdout)
ARTIFACT_NAME_PKG=$(mvn org.apache.maven.plugins:maven-help-plugin:3.1.1:evaluate -Dexpression=project.packaging -q -DforceStdout)

# compute the app name with packaging
APP_NAME="$ARTIFACT_NAME.$ARTIFACT_NAME_PKG"
```


https://github.com/redhat-developer-demos/quarkus-java-builder/blob/master/bin/maven-buildah.sh#L15-L19

Reply:
Resolving this, as I've now been able to figure out the right magic combination which does the trick to make this work.